### PR TITLE
Bug 1198165 - don't pause the video when the user seeks.

### PR DESCRIPTION
**Pull request by davidflanagan**
This prevents a music app in the background from resuming.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f98ceb5f30c7ba1a93072c435962b5c4cde9c63d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=59c35f10c6c2fc28efb6e9d8e73fec9c7a6dc392

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=27719283b34a343f717b92dea9a4b534d2612d23

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3a4e74afbc82b8dc5ca6b2f473d143c2361482e7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9b7d43a800849e6731fc87924ae0ebf0a6a99059

File: apps/video/js/video.js [hunk 1]
```diff
@@ -1313,10 +1310,12 @@ function handleSliderTouchEnd(event) {
 
   dom.playHead.classList.remove('active');
 
+  // We didn't move the play head while the user's finger was on it
+  // so now it may be in the wrong place.
+  updateVideoControlSlider();
```
Comment (russnicoletti) [inline apps/video/js/video.js]: Is this change is left over from when the patch was not using the 'seeker'? Now that you are seeking while the slider is being dragged it seems this code is not needed.

File: apps/video/js/view.js [hunk 2]
```diff
@@ -497,10 +496,12 @@
 
     dom.playHead.classList.remove('active');
 
+    // We didn't move the play head while the user's finger was on it
+    // so now it may be in the wrong place.
+    updateSlider();
+
```
Comment (russnicoletti) [inline apps/video/js/view.js]: Is it necessary to call updateSlider here since the play head is being moved as the slider is being dragged?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=79e628de374e353b50730ea6c5d45439668f1656

File: apps/video/js/video.js (hunk 1 above)
Comment (davidflanagan) [inline apps/video/js/video.js]: The issue isn't related to seeker. It is that we don't update the play head position on timeupdate events when the user's finger is down. Now that we're playing during the drag, it is possible for the play head to get behind the actual time.  And it is a serious bug if the user holds their finger down until the video stops. Then the play head is stuck at the wrong place.  

See the `if (!dragging)` statement in updateVideoControlSlider().

So yes, this is still needed.

File: apps/video/js/view.js (hunk 2 above)
Comment (davidflanagan) [inline apps/video/js/view.js]: yes, necessary. updateSlider() has the same `if (!dragging)` code, so we've got to call it again when the dragging ends.

**Comment by davidflanagan** [comment]
The tests didn't run. I'm going to close and re-open the PR to get them to run again.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=79e628de374e353b50730ea6c5d45439668f1656

**Comment by davidflanagan** [comment]
Again!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=79e628de374e353b50730ea6c5d45439668f1656

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=79e628de374e353b50730ea6c5d45439668f1656

**Pull request merged**